processors/config/recoTuple_cfg.py

```diff
@@ -138,6 +138,15 @@
 mcpart.parameters["mcPartCollLcio"] = 'MCParticle'
 mcpart.parameters["mcPartCollRoot"] = 'MCParticle'
 
+#FinalStateParticleProcessor
```

Comment:
What I want to do with this file is add FSPs, and also add a new option that will add hits or not depending on the option given. We can discuss in person how we want to deal with having the hits for tracks vs fsp, or both.

Reply:
Done